src/test/resources/hudson/plugins/active_directory/ActiveDirectorySecurityRealmFipsAndForceLdapEnabledTest/testRequireTlsForWarnings/config.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<hudson>
+  <disabledAdministrativeMonitors/>
+  <version>1.0</version>
+  <numExecutors>2</numExecutors>
+  <mode>NORMAL</mode>
+  <useSecurity>true</useSecurity>
+  <authorizationStrategy class="hudson.security.AuthorizationStrategy$Unsecured"/>
+  <securityRealm class="hudson.plugins.active_directory.ActiveDirectorySecurityRealm" plugin="active-directory@2.0">
+    <domains>
+      <hudson.plugins.active__directory.ActiveDirectoryDomain>
+        <name>example.com</name>
+      </hudson.plugins.active__directory.ActiveDirectoryDomain>
+    </domains>
+    <site>site</site>
+    <bindName>bindUser</bindName>
+    <bindPassword>Uv8/yXQyHku/Bx5Shl4TsTa4MJDBxBiJTSIqpEt2+DA=</bindPassword>
```

Comment:
doesn't seem a real `Secret` (whose format would be like ${Uv8/yXQyHku/Bx5Shl4TsTa4MJDBxBiJTSIqpEt2+DA=}, so the `secrets/master.key` might not be needed. Could you please check?